docs/manual/developer/06_contributing_with_content.md

```diff
@@ -1577,6 +1577,33 @@ the following to `rule.yml`:
 
 -   Languages: Ansible, Bash, OVAL, Kubernetes
 
+#### sudo_defaults_option
+This template ensures a sudo `Defaults` options is enabled in `/etc/sudoers`
+or in `/etc/sudoers.d/*`.
+The remediations add the `Defaults` option  to `/etc/sudoers` file.
```

Comment:
I deviated a bit from other templates format of all bullet points.
I think it is more readable this way, thoughts?

Reply:
The context probably changed, as it looks the same as documentation for other templates, and it is very well readable.